### ISSUE TITLE
Updated badges label on README.md

### ISSUE THREAD

**Issue by enricobenedos** [enricobenedos/GPXReaderLib#3]
Need to update README badges label from ".NET CORE" to `develop` and `master`

**Comment by huseyinsimsekk** [comment]
I can do :) Please assign me

**Comment by enricobenedos** [comment]
Hi @huseyinsimsekk, nice to meet you. I will accept your PR when it will be ready 😃 

Just to be more clear these one are the badges to fix:
![image](https://user-images.githubusercontent.com/12233746/95834447-42370400-0d3d-11eb-958a-889f6ba1aa2c.png)

**Comment by huseyinsimsekk** [comment]
Hi @enricobenedos . Nice to meet you, too. First I think I will just change readme file :D but its not work. I google it and than I found a solution that creates new Github actions. I created to two action called 'Master' and 'Develop' in my fork repository. Than I changed readme file but I also changed link with my username because it is neccessary. Finally it looks like: 
 
![resim](https://user-images.githubusercontent.com/29932111/95845134-77018600-0d52-11eb-8362-e248c09c9cd3.png)
